app/routes/events/components/EventAttendeeStatistics.tsx

```diff
@@ -406,11 +406,14 @@ const EventAttendeeStatistics = ({
             label={'Gruppetilhørighet'}
             distributionData={groupDistribution}
           />
-
-          <h4>Påmeldinger og avmeldinger per dag</h4>
+        </div>
+      )}
+      <Card className={styles.graphCard}>
+        <h4 className={styles.graphCardTitle}>
+          Påmeldinger og avmeldinger per dag
+        </h4>
```

Comment:
I just realized it would be nice to use the built in card header instead
```suggestion
        <Card.Header>
          Påmeldinger og avmeldinger per dag
        </Card.Header>
```

I suppose you won't need the `graphCardTitle` class then

Reply:
Done @ivarnakken